gorjana/flows/sales-orders/blanket-customer.md

```diff
@@ -0,0 +1,119 @@
+---
+description: >-
+  Explore how gorjana handles unregistered POS Cash Order by mapping them to a
+  blanket customer in HotWax for seamless order processing and integration with
+  NetSuite.
+---
+
+# Blanket Customer
+
+## Blanket Customer
+
+When creating POS Cash Order in store, high traffic can lead to customers not being registered against the sale. Though most of the time the staff will log the customer details right after entering the order, it's possible that the customer is not added at all. In this case the order is mapped to a blanket customer in HotWax. When posting this order to NetSuite, gorjana has a many blanket customer's that orders without customers should be mapped to.
+
+### How gorjana Manages the Blanket Customers?
+
+In case of blanket customers, gorjana has a default customer in Netsuite for each store.
+
+While syncing the POS sale orders with blanket customers from HotWax to NetSuite, we need to send the `NETSUITE_CUSTOMER_ID` for the store where the order was completed.
+
+### How this is setup in HotWax?
+
+For orders lacking specific customer details, they are associated with the `_NA_` party_ID in HotWax. However, to ensure these orders can be synced with NetSuite, the corresponding customer must have a `NETSUITE_CUSTOMER_ID` in NetSuite.
+
+The `_NA_` party_ID doesn't have a `NETSUITE_CUSTOMER_ID`, making such orders ineligible for synchronization. To address this, it's crucial to create a party identification `_NA_` ID with the type `NETSUITE_CUSTOMER_ID` and assign it a dummy value. This ensures that orders with an `_NA_` party_ID are eligible for POS order feed synchronization.
```

Comment:
To address this, it's crucial to create a party identification `_NA_` ID with the type `NETSUITE_CUSTOMER_ID` and assign it a dummy value.  -> To address this, it's crucial to create a party identification for `_NA_` party with the type `NETSUITE_CUSTOMER_ID` and assign it a dummy value.

Reply:
Changes updated